airbyte-integrations/connectors/destination-mssql-v2/src/test-integration/kotlin/io/airbyte/integrations/destination/mssql/v2/MSSQLWriterTest.kt

```diff
@@ -151,5 +268,36 @@ internal class StandardInsert :
         fun beforeAll() {
             MSSQLContainerHelper.start()
         }
+
+        /** Builds a map of overrides for the test container environment. */
+        private fun buildOverridesForTestContainer(): MutableMap<String, String> {
+            return mutableMapOf("host" to MSSQLContainerHelper.getHost()).apply {
+                MSSQLContainerHelper.getPort()?.let { port -> put("port", port.toString()) }
+            }
+        }
+    }
+}
+
+internal class BulkInsert :
+    MSSQLWriterTest(
+        configPath = Path.of(CONFIG_FILE),
+        configUpdater = FakeConfigurationUpdater,
```

Comment:
@subodh1810 Looks like we are not using the real updater that runs the commands for the bulk insert to work?

Reply:
Yeah we dont need the updater cause we are not mutating the config in any way. the secret that lives in GSM is actual configuration that the connector can work with

Reply:
+1.  I confused myself.  We are still starting the MSSQL container via the helper, which runs the SQL commands to set up the data source for bulk insert, correct?

Reply:
No we are running the BULK upload tests against the sql server instance that lives in azure cloud